crates/recursor/src/recursor.rs

```diff
@@ -319,6 +319,22 @@ impl Recursor {
                 handle,
                 record_cache,
             } => {
+                if let Some(Ok(lookup)) = record_cache.get(&query, request_time) {
+                    let all_records_have_undergone_dnssec_validation = lookup
```

Comment:
Can we just call it `all_validated`?

Reply:
I wasn't quite sure about the naming here because the outcome of the DNSSEC validation process is to change the `proof` field of each record in the response to something other than `Indeterminate` and "all validated" to me sounds like "all `proof`s are the `Secure` variant" which may not always be the case and it's also not what we check here.

maybe we can use "all_determined"?

Reply:
Maybe `none_indeterminate`?